SPHINXsys/src/shared/particle_dynamics/solid_dynamics/thin_structure_math.cpp

```diff
@@ -57,7 +57,7 @@ namespace SPH
 			return drotation_matrix_dt * initial_vector;
 		}
 		//=================================================================================================//
-		Vec3d getVectorChangeRateAfterThinStructureRotation(const Vec3d& initial_vector, const Vec3d& rotation_angles, const Vec3d& angular_vel)
+		Vec3d getVectorChangeRateAfterThinStructureRotation(const Vec3d &initial_vector, const Vec3d &rotation_angles, const Vec3d &angular_vel)
```

Comment:
@Xiangyu-Hu these math functions should be unit tested. Please add unit tests for these  @mrezavandVirtonomy